plugins/discover.yaml

```diff
@@ -0,0 +1,111 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: discover
+spec:
+  version: v0.1.5
+  homepage: https://github.com/mateimicu/kdiscover
+  shortDescription: "Discover Kubernetes clusters and export configs"
+  description: |
+    Discover all the cluster from a provider (AWS, GCP, Azure, etc)
+    and manage there credentials.
```

Comment:
"There" seems grammatically incorrect.

Reply:
I reworded it a bit to match the `shortDescription`, sounds good?